python/ray/autoscaler/command_runner.py

```diff
@@ -117,13 +125,16 @@ def __init__(self, log_prefix, namespace, node_id, auth_config,
         self.namespace = namespace
         self.kubectl = ["kubectl", "-n", self.namespace]
 
-    def run(self,
+    def run(
+            self,
             cmd=None,
             timeout=120,
             exit_on_fail=False,
             port_forward=None,
             with_output=False,
-            **kwargs):
+            run_env="auto",  # Unused argument.
+            ssh_options_override_ssh_key="",  # Unused argument.
```

Comment:
Should it raise an error if it's set then?

Reply:
No, definitely not. It is set in NodeUpdater for all `CommandRunners`, but only used in `DockerCommandRunner`

Reply:
I see, is a more accurate name something like "force_run_outside_container"?

Reply:
run_env is the name of the entry in the cluster.yaml config and also was renamed from docker to run_env. I guess this is a separate PR.

Reply:
Just to make it clear, run_env was often passed to these functions while they were not used. After I got confused for 5 minutes I thought it would be helpful to add a comment when it is used or not.